src/components/autocomplete/js/autocompleteController.js

```diff
@@ -466,9 +466,9 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
         select(ctrl.index);
         break;
       case $mdConstant.KEY_CODE.ESCAPE:
+        event.preventDefault(); // Prevent browser from always clearing input
```

Comment:
@clshortfuse Okay, thanks for the clarification.

Quick note: the esc key still blurs the input even with `md-escape-options="none"`. I would assume that `none` would specifically disable the blurring too, but I honestly don't know how it was implemented originally.

Can you take a quick look at that and see if it's related to this issue?